io.github.serial_studio.Serial_Studio.yaml

```
@@ -0,0 +1,46 @@
+app-id: io.github.serial_studio.Serial_Studio
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: serial-studio
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --device=all
```

```suggestion
  - --device=all
```

`dri` in included in `all`

---

OK, get it